metriq_gym/dispatch_qv.py

```diff
@@ -16,24 +16,30 @@
 def main():
     args = parse_arguments()
 
+    try:
+        params = load_and_validate(args.input_file)
+    except ValidationError as e:
+        logging.error(f"JSON validation error: {e.message}")
+        return 1
```

Comment:
If this exception is hit, I think the `return 1` might be misleading (and also unnecessary) as hitting the exception case should automatically return.

Reply:
I agree it's redundant, I will remove.